features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/Debouncer.kt

```diff
@@ -53,24 +54,31 @@ internal class Debouncer(
             val endTimeInNano = System.nanoTime()
             timeBank.consume(endTimeInNano - startTimeInNano)
         } else {
-            // TODO RUM-5188 report with telemetry about the frames skipped
-            internalLogger.log(
-                InternalLogger.Level.INFO,
-                InternalLogger.Target.MAINTAINER,
-                messageBuilder = {
-                    MSG_FRAME_SKIP
-                }
-            )
+            logSkippedFrame()
         }
     }
 
+    private fun logSkippedFrame() {
+        val rumFeature = sdkCore.getFeature(Feature.RUM_FEATURE_NAME) ?: return
+        val telemetryEvent =
+            mapOf(
+                TYPE_KEY to TYPE_VALUE,
+                MESSAGE_KEY to { },
```

Comment:
what is the point of passing empty lambda?

Reply:
removed